Rakefile

```diff
@@ -6,7 +6,14 @@ require 'puppetlabs_spec_helper/rake_tasks'
 require 'puppet-syntax/tasks/puppet-syntax'
 require 'puppet_blacksmith/rake_tasks' if Bundler.rubygems.find_name('puppet-blacksmith').any?
 require 'github_changelog_generator/task' if Bundler.rubygems.find_name('github_changelog_generator').any?
-require 'puppet-strings/tasks' if Bundler.rubygems.find_name('puppet-strings').any?
+
+if Bundler.rubygems.find_name('puppet-strings').any?
+  require 'puppet-strings/tasks'
+  # strings:validate:reference would raise an error if REFERENCE.md is not
+  # up-to-date, but we only want to be sure that all documentation is in place
+  # and ensure it is kept up-to-date.
+  Rake::Task[:validate].enhance ['strings:generate:reference']
```

Comment:
I wonder if this belongs in https://github.com/puppetlabs/puppetlabs_spec_helper and in particular https://github.com/puppetlabs/puppetlabs_spec_helper/blob/7aae63b2f7177abd1c39ddba06ace7563079827f/lib/puppetlabs_spec_helper/rake_tasks.rb#L32-L36

In fact, this whole require is redundant, just like some other lines are (line 6 for example)

Reply:
This change will also be overwritten by the next pdksync as it is not included within the templates.